client/beta/top-links.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Link } from 'react-router'
 import styles from './beta.css'
 
-const STARCRAFT_DOWNLOAD_URL = 'https://us.battle.net/account/download/?show=classic'
+export const STARCRAFT_DOWNLOAD_URL = 'https://us.battle.net/account/download/?show=classic'
```

Comment:
Little weird to import this from here, I think we should move this to its own file